tests/integration/hub_usage/test_hub_usage.py

```diff
@@ -148,3 +148,11 @@ def test_hub_build_push_push_again(monkeypatch, mocker):
         # try and push same version again should fail with `--no-overwrite`
         args = set_hub_build_parser().parse_args([str(cur_dir) + '/hub-mwu', '--push', '--host-info', '--no-overwrite'])
         HubIO(args).build()
+
+
+@pytest.mark.timeout(360)
+@pytest.mark.parametrize('dockerfile_path', [os.path.join(cur_dir, 'hub-mwu-multistage'), 'hub-mwu-multistage'])
```

Comment:
Try replacing this with
```python
hub_mwu_ms_abspath = os.path.join(cur_dir, 'hub-mwu-multistage')

@pytest.mark.timeout(360)
@pytest.mark.parametrize('dockerfile_path', [hub_mwu_ms_abspath, os.path.relpath(hub_mwu_ms_abspath, os.getcwd())])
```

I think now the CI for relative path is failing due to difference between `cur_dir` and `cwd`, this should fix it.

Reply:
fixed